cmd/agent/subcommands/launchgui/command.go

```diff
@@ -64,11 +64,11 @@ func launchGui(config config.Component, _ *cliParams, _ log.Component) error {
 	}
 
 	// Open the GUI in a browser, passing the authorization tokens as parameters
-	err = open("http://127.0.0.1:" + guiPort + "/auth?intent=" + string(intentToken))
```

Comment:
Can you add comments in the code to make it clear that we use `localhost` instead of `127.0.0.1` on purpose ?